app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/controllers/AdditionalIncomeSourcesController.scala

```
@@ -59,7 +59,7 @@ class AdditionalIncomeSourcesController @Inject()(view: AdditionalIncomeSourcePa
       answers => {
         val sessionId = request.session.get("sessionId").getOrElse("")
```

I don't thing a blank `sessionId` is acceptable. We should throw an error if that occurs.